docs/pages/includes/config-reference/auth-service.yaml

```diff
@@ -58,7 +58,6 @@ teleport:
         # consume the retention period via a query parameter in the audit_events_uri. See the examples below
         # for how to configure the retention period for other backends.
         # Firestore: firestore://events_table_name?eventRetentionPeriod=10d
-        # Postgres: postgresql://user_name@database-address/teleport_audit?retention_period=10d
         retention_period: 365d
```

Comment:
The correct syntax for pgevents would be something like `postgresql://user_name@database-address/teleport_audit#retention_period=240h` as mentioned in https://goteleport.com/docs/reference/backends/#postgresql , I don't know if we should mention that here or not.

Reply:
@programmerq Does `postgresql://user_name@database-address/teleport_audit#retention_period=240h` work in your test environment?

Reply:
I just successfully tried this in a lab, and updated the doc PR to point out the fragment.